npm/package-lock.json

```diff
@@ -1,20 +1,55 @@
 {
   "name": "npm-test",
   "version": "1.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 3,
```

Comment:
Did you do this intentionally? 

Our support for NPM 9 is not there yet from what I understand 

See https://github.com/dependabot/dependabot-core/blob/main/npm_and_yarn/Dockerfile#L13